### ISSUE TITLE
EarlyExit does not report useless else

### ISSUE THREAD

**Issue by momala454** [slevomat/coding-standard#1496]
The issue reported in 2020 seems to still happen in 8.8.0
https://github.com/slevomat/coding-standard/issues/967

```php
if (1) {
            throw new \Exception('');
        } elseif (1) {
            echo 'bla';
        } else {
            echo 'hello';
        }

        if (1) {
            throw new \Exception('');
        } else {
            echo 'hello';
        }
```

Only the useless `elseif` is detected, not the two useless else

```
-----------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------------
 94 | ERROR | [x] Use "if" instead of "elseif".
-----------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-----------------------------------------------------------------------------------------
```

**Comment by kukulich** [comment]
Please try to prepare PR with failing test. I'm not able to reproduce it.

**Comment by momala454** [comment]
@kukulich , added a pull request https://github.com/slevomat/coding-standard/pull/1520
